src/foam/nanos/crunch/Refinements.js

```diff
@@ -31,18 +31,9 @@ foam.CLASS({
       createVisibility: 'HIDDEN',
       updateVisibility: 'RO',
       projectionSafe: false,
-      tableCellFormatter: function(value, obj) {
-        obj.userDAO
-          .where(obj.EQ(foam.nanos.auth.User.ID, value))
-          .limit(1)
-          .select(obj.PROJECTION(foam.nanos.auth.User.LEGAL_NAME))
-          .then(function(result) {
-            if ( ! result || result.array.size < 1 || ! result.array[0]) {
-              this.add(value);
-              return;
-            }
-            this.add(result.array[0]);
-          }.bind(this));
+      tableCellFormatter: {
+        class: 'foam.u2.view.ReferencePropertyCellFormatter',
```

Comment:
Why is this needed? Why not just use the existing ReferenceToSummaryCellFormatter, which will call to toSummary() method on User, and the first line of that is:
        if ( this.legalName ) return this.legalName;

Reply:
ReferenceToSummaryCellFormatter doesn't show the value if the object is not found. 

Now updating ReferenceToSummaryCellFormatter https://github.com/kgrgreer/foam3/pull/2848/commits/0143a0669755dcb81dbf9fa64618fdea4767ea24 @kgrgreer @sarthak-marwaha

Reply:
Better to fix ReferenceToSummaryCellFormatter then.